tab_pastevents.md

```diff
@@ -12,13 +12,35 @@ tags: cincinnati
 ## Past Events
 <hr>
 
+## November 2022 Meeting
+
+### Security Champions Program
+
+**Discussion Abstract:** A security champions program is unique to the organization and their development teams. As application security teams begin to look towards these types of programs, it's important to have a good understanding of the complexities and goals a program like this will have, along with the relationship your security team has with your development team. In this talk we'll go through what growing and maintaining an effective program looks like, along with some of obstacles and ways to overcome them.
+
+**Speaker Biography:** Jay Simmons is a senior application security analyst at Great American Insurance Group who is responsible for their security champions program. 
+
+**Presentation:*
+[OWASP Cinci - November 2022 - Security Champions Program](https://owasp.org/www-chapter-cincinnati/assets/presentations/Maintaining%20a%20Security%20Champions%20Program.pdf)
```

Comment:
Will this link automatically work once this PR is merged?

Reply:
I think it would, I tested this with the current assets to verify that url path matches, but I can use relational links if that would make more sense.

Reply:
Updated pull request to include relative links instead of strict links and also updated gemfile and gitignore to support local development rendering.